core/src/main/java/by/ilyin/core/controller/AuthCoreController.java

```diff
@@ -0,0 +1,48 @@
+package by.ilyin.core.controller;
+
+import by.ilyin.core.dto.SignInDTO;
+import by.ilyin.core.entity.CustomJWT;
+import by.ilyin.core.entity.CustomUser;
+import by.ilyin.core.service.auth.AuthService;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.Set;
+
+@RestController
+@RequiredArgsConstructor
+@RequestMapping("/api")
+public class AuthCoreController {
+
+    private final AuthService authService;
+
+    @PostMapping("/sign-in")
+    public CustomUser signIn(@RequestBody SignInDTO signInDTO) {
+        return authService.getUserByCredentials(signInDTO);
+    }
+
+    @PostMapping("/logout")
+    public Set<CustomJWT> logout(@RequestParam(value = "userId") Long userId) {
+        return authService.logoutProcess(userId);
+    }
+
+    @GetMapping("/jwt")
+    public Set<CustomJWT> getUpToDateBlackList() {
+        return authService.getJwtBlackList();
+    }
+
+    @PostMapping("/jwt")
```

Comment:
@YauhenIlyin where did you find it?

Reply:
by agreement, I implemented it until I received exact instructions